scala-package/core/src/main/scala/org/apache/mxnet/NDArray.scala

```diff
@@ -89,7 +89,10 @@ object NDArray extends NDArrayBase {
         output match {
           case nd: NDArray => (Array(nd), Array(nd.handle))
           case ndFuncRet: NDArrayFuncReturn => (ndFuncRet.arr, ndFuncRet.arr.map(_.handle))
-          case ndArr: Seq[NDArray] => (ndArr.toArray, ndArr.toArray.map(_.handle))
+          case ndArr: Seq[NDArray @unchecked] =>
+            if (ndArr.head.isInstanceOf[NDArray]) (ndArr.toArray, ndArr.toArray.map(_.handle))
```

Comment:
Why `@unchecked` and again the need to check if the head is a Instance of NDArray?

Reply:
Please check the Stackoverflow link above, the problem here is Scala cannot identify the type T of a Seq[T]. We need to break it and check inside to make sure it match our needs.

Reply:
Thank you @lanking520, would be great if you could add the link as a comment above the code.